Week-3/Responsiveness/Media queries for a vacation website/index.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+   <title>Media Query Lab</title>
+   <link rel="stylesheet" href="./styles.css">
+   <meta name="viewport" content="width=device-width, initial-scale=1">
+</head>
+<body>
+   <header>
+      <h1>Mediterranian Vacations</h1>
+   </header>
+   <nav>
+      <ul>
+         <li><a href="#featured">Featured Packages</a></li>
+         <li><a href="#exotic">Exotic Vacations</a></li>
+         <li><a href="#historic">Historic Locations</a></li>
+      </ul>
+   </nav>
+   <div class="benefits-container">
+      <div class="benefits-inner-container">
+         <div>
+            <img src="checkmark.png" alt="checkmark">
+         </div>
+         <div class="benefit">
+            <strong>Affordable!</strong> Our vacation packages give you
+            the most bang for your buck.
+         </div>
+      </div>
+      <div class="benefits-inner-container">
+         <div>
+            <img src="checkmark.png" alt="checkmark">
+         </div>
+         <div class="benefit">
+            <strong>Quality!</strong> We use only the finest lodgings
+            and qualified tour guides.
+         </div>
+      </div>
+      <div class="benefits-inner-container">
+         <div>
+            <img src="checkmark.png" alt="checkmark">
+         </div>
+         <div class="benefit">
+            <strong>Cancel Anytime!</strong> You may cancel your trip
+            at any time for a full refund.
+         </div>
+      </div>
+   </div>
+   <main>
+      <section id="featured">
+         <h2>Featured Packages</h2>
+         <div class="vacation-container">
+            <img src="egypt_600.jpg" alt="Great Sphyinx and pyramid">
```

Comment:
always use Relative Path when linking any file, image 

this goes for the entire file